.editorconfig

```
@@ -187,17 +182,29 @@ fsharp_max_function_binding_width=80
 # default 50
 fsharp_max_dot_get_expression_width=80
 
-# whether open/close brackets go on same colum
-# false: type Range =
-#            { From: float
-#              To: float }
-# true:  type Range =
-#           {
-#               From: float
-#               To: float
-#           }
-# default false
-fsharp_multiline_block_brackets_on_same_column=false
+# whether open/close brackets go on the same column
+# cramped:    type Range =
+#               { From: float
+#                 To: float }
+# aligned:    type Range =
+#                 {
+#                     From: float
+#                     To: float
+#                 }
+# stroustrup: type Range = {
+#                 From: float
+#                 To: float
+#             }
+# default cramped
+fsharp_multiline_bracket_style=stroustrup
```

Isn't it supposed to be? :thinking:
```suggestion
fsharp_multiline_bracket_style=cramped
```

---

I prefer `stroustrup` as `cramped` is hard to manage